### PR TITLE
Add option to change cycle countdown.

### PR DESCRIPTION
**Pull request by benrobson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,15 +61,15 @@ Here's a basic example of what map configuration files look like:
...
+1. Clone the repo to your computer. `git clone https://github.com/Warzone/TGM`
 
 2. Compile CraftBukkit by downloading and running the BuildTools jar from [here](https://hub.spigotmc.org/jenkins/job/BuildTools). Make sure you include the `-compile craftbukkit` flag when compiling it, otherwise you might receive errors when later compiling the plugin.
 
 3. Make sure you have Maven installed and run `mvn clean install` in the top-level folder to generate the required libraries. You can then download the jar from the `target` folder.
 
 ## Developer Tips
 
-1. We use [Lombok](https://projectlombok.org/). Make sure you have the Lombok plugin installed on your preferred IDE.
+1. We use [Lombok](https://projectlombok.org/). Make sure you have the Lombok plugin installed on your preferred IDE, otherwise you cannot compile.
```
Comment (ronthecookie) [inline README.md:72]: This isn't true, you can still compile directly with Maven even without the Lombok plugin installed on your IDE.

File: README.md [hunk 2]
```diff
@@ -61,15 +61,15 @@ Here's a basic example of what map configuration files look like:
 
 ## Compiling
 
-1. Clone the repo to your computer.
+1. Clone the repo to your computer. `git clone https://github.com/Warzone/TGM`
```
Comment (ronthecookie) [inline README.md:64]: Docs changes should be allotted to a different PR if they aren't relevant to the main change.

File: TGM/pom.xml [hunk 3]
```diff
@@ -88,4 +88,4 @@
         </dependency>
     </dependencies>
 
-</project>
+</project>
```
Comment (ronthecookie) [inline TGM/pom.xml:91]: Nitpick but no trailing newline?

File: pom.xml [hunk 4]
```diff
@@ -132,4 +132,4 @@
         </dependency>
     </dependencies>
 
-</project>
+</project>
```
Comment (ronthecookie) [inline pom.xml:135]: Nitpick but no trailing newline? Why touch this in the first place? It's completely unrelated to your main change.

File: TGM/src/main/resources/config.yml [hunk 5]
```diff
@@ -38,6 +38,9 @@ map:
   # Store the game matches in the `matches` folder found in the server root directory.
   # Default: false
   save-matches: false
+  # When the match ends, how long to take to cycle to the next map.
+  # Default: 10
```
Comment (ronthecookie) [inline TGM/src/main/resources/config.yml:42]: Where is this default defined? I can't find any definition for this value of `10`.

**Review by ronthecookie — CHANGES_REQUESTED** [review]

File: TGM/src/main/resources/config.yml (hunk 5 above)
Comment (benrobson) [reply TGM/src/main/resources/config.yml:42]: I put this here because all of the other options have a default option, so to fit, that's what I did.

**Review by ronthecookie — APPROVED** [review]
Still no trailing newlines on the poms but small enough that it can pass.

**Comment by BennyDoesStuff** [comment]
I tested this PR on my server and somehow the default value of 10 worked, and it isn't even defined anywhere in the code. Not even gonna ask questions
